main/java/TestForJsonParserClassWriteTo.java

```diff
@@ -0,0 +1,29 @@
+import org.junit.jupiter.api.Test;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.Arrays;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class TestForJsonParserClassWriteTo extends BeforeAfter {
+
+    @Test
+    public void scenarioWriteToValidData() {
```

Comment:
Give me example when the test will fail and catch a bug

Reply:
I expect that if there is no "resources" folder in place the test case will fail.
Frankly speaking, I was trying to add a test with invalid data where I was removing this folder previously copying files from it to a temporary folder (and I was restoring this folder with its files in "After" part) but it turned out that I cannot check not throwable exception and I removed this check.
Do I need to add anything else here? Or it was just a question?

Reply:
I got what you asked about finally =))
I didn't notice that my assertTrue(true); will always make test pass. If your question was aimed to that then I found it and changed (added condition when assertion will be true and otherwise it will fail).